### PR TITLE
i18n(zh-CN): Translate client-side scripts and nav strings

### PR DESCRIPTION
**Pull request by zihan-ch**
#### What kind of changes does this PR include?

- Translated content

#### Description

Add translations

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Thanks @zihan-ch! Would you mind fixing the broken links? I think everywhere you used `zh-CN` should be `zh-cn` all lowercase.

File: src/pages/zh-cn/guides/client-side-scripts.mdx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+<!-- 也适用于本地 TypeScript 文件 -->
+<script src="./script-with-types.ts"></script>
+```
+
+#### 加载外部脚本
+
+**何时使用：** 如果您的 JavaScript 文件位于 `public/` 或 CDN 上。
+
+To load scripts outside of your project's `src/` folder, include the `is:inline` directive. This approach skips the JavaScript processing, bundling, and optimizations that are provided by Astro when you import scripts as described above.
```
Comment (delucis) [inline src/pages/zh-cn/guides/client-side-scripts.mdx:77]: Looks like the English text got left in here — the translation is below.

```suggestion
```

File: src/pages/zh-cn/guides/client-side-scripts.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+```
+
+## 常见脚本模式
+
+### 处理 `onclick` 和其他事件
+
+Some UI frameworks use custom syntax for event handling like `onClick={...}` (React/Preact) or `@click="..."` (Vue). Astro follows standard HTML more closely and does not use custom syntax for events.
+
+Instead, you can use [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) in a `<script>` tag to handle user interactions.
+
```
Comment (delucis) [inline src/pages/zh-cn/guides/client-side-scripts.mdx:95]: Same here, we can remove the English original text.

```suggestion
```

File: src/pages/zh-cn/guides/client-side-scripts.mdx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+  buttons.forEach((button) => {
+    button.addEventListener('click', () => {
+      alert('按钮被点击了！');
+    });
+  });
+</script>
+```
+
+:::note
+如果您在一个页面上有多个 `<AlertButton />` 组件，Astro 将不会多次运行该脚本。 脚本是捆绑在一起的，每页只包含一次。 使用 querySelectorAll 确保此脚本将事件侦听器附加到页面上找到的具有 alert 类的每个按钮。
```
Comment (delucis) [inline src/pages/zh-cn/guides/client-side-scripts.mdx:117]: We can use code backticks here:

```suggestion
如果您在一个页面上有多个 `<AlertButton />` 组件，Astro 将不会多次运行该脚本。 脚本是捆绑在一起的，每页只包含一次。 使用 `querySelectorAll` 确保此脚本将事件侦听器附加到页面上找到的具有 `alert` 类的每个按钮。
```

**Review by delucis — COMMENTED** [review]
Thanks for this @zihan-ch. I left a few small suggestions, but looks great! 🙌

**Comment by zihan-ch** [comment]
Thanks for reviewing @delucis! I am really careless🤪

**Review by delucis — APPROVED** [review]
All things you'll remember to double check next PR 😉

Thanks again — LGTM! 🚀

**Pull request merged**